ethclient/ethclient_test.go

```
@@ -222,7 +222,7 @@ func generateTestChain() (*core.Genesis, []*types.Block) {
 		g.OffsetTime(5)
 		g.SetExtra([]byte("test"))
 	}
-	gblock := genesis.ToBlock(db)
+	gblock, _ := genesis.ToBlock(db)
```

we shouldn't ignore errors

---

It would have panicked but I'll make it more explicit.